lvmd/vgservice.go

```diff
@@ -111,14 +155,53 @@ func (s *vgService) send(server proto.VGService_WatchServer) error {
 		if err != nil {
 			return err
 		}
-		if dc.Default {
-			res.FreeBytes = vgFree
+
+		switch dc.Type {
+		case TypeThick:
+			if dc.Default {
+				res.FreeBytes = vgFree
+			}
+			res.Items = append(res.Items, &proto.WatchItem{
+				DeviceClass: dc.Name,
+				FreeBytes:   vgFree,
+				SizeBytes:   vgSize,
+			})
+		case TypeThin:
+			tpi := &proto.ThinPoolItem{}
+			pool, err := vg.FindPool(dc.ThinPoolConfig.Name)
+			if err != nil {
+				return status.Error(codes.Internal, err.Error())
+			}
+			tpu, err := pool.Free()
+			if err != nil {
+				return status.Error(codes.Internal, err.Error())
+			}
+			opb := uint64(math.Ceil(dc.ThinPoolConfig.OverprovisionRatio*float64(tpu.SizeBytes))) - tpu.VirtualBytes
```

Comment:
```suggestion
			opb := uint64(math.Ceil(dc.ThinPoolConfig.OverprovisionRatio*float64(tpu.FreeBytes)))
```

Reply:
- will be tracked under https://github.com/topolvm/topolvm/pull/448#discussion_r838056977